services/rating/src/main/java/com/eshop/rating/infrastructure/SpringQueryBus.java

```diff
@@ -0,0 +1,29 @@
+package com.eshop.rating.infrastructure;
+
+
+import com.eshop.rating.application.querybus.Query;
+import com.eshop.rating.application.querybus.QueryBus;
+import com.eshop.rating.application.querybus.QueryHandler;
+import lombok.RequiredArgsConstructor;
+import org.springframework.beans.factory.ObjectProvider;
+import org.springframework.stereotype.Component;
+
+//TODO IGI move this to a infrastructure module
```

Comment:
what about this todo?

Reply:
Yeah just a suggestion that may become a discussion topic. What do you think of introducing some common infrastructure module for all message-based configuration so it can be imported whenever it's needed ? @hdimitrieski @velinovskav